Legion/CourtOfStars/Trash.lua

```diff
@@ -0,0 +1,175 @@
+-- Initialization
+--
+
+function mod:GetOptions()
+	return {
+		209027, -- Quelling Strike (Duskwatch Guard)
+		209033, -- Fortification (Duskwatch Guard)
+		{209036, "SAY", "FLASH"}, -- Throw Torch (Duskwatch Sentry)
+		225100, -- Charging Station (Guardian Construct)
+		209495, -- Charged Smash (Guardian Construct)
+		209512, -- Disrupting Energy (Guardian Construct)
+		{209413, "SAY", "FLASH"}, -- Suppress (Guardian Construct)
+		211464, -- Fel Detonation (Felbound Enforcer)
+		211391, -- Felblaze Puddle (Legion Hound)
+		214692, -- Shadow Bolt Volley (Gerenth the Vile)
+		214688, -- Carrion Swarm (Gerenth the Vile)
+		{214690, "SAY", "FLASH"}, -- Cripple (Gerenth the Vile)
+		207979, -- Shockwave (Jazshariu)
+		209378, -- Whirling Blades (Imacu'tya)
+		{207980, "SAY", "FLASH"}, -- Disintegration Beam (Baalgar the Watchful)
+		211299, -- Searing Glare (Watchful Inquisitor)
+		212784, -- Eye Storm (Watchful Inquisitor)
+		211401, -- Drifting Embers (Blazing Imp)
+		212031, -- Charged Blast (Bound Energy)
+		{211470, "SAY", "FLASH"}, -- Bewitch (Shadow Mistress)
+		{211473, "SAY", "FLASH"}, -- Shadow Slash (Shadow Mistress)
+		209485, -- Drain Magic (Arcane Manifestation)
+		209477, -- Wild Detonation (Mana Wyrm)
+		209410, -- Nightfall Orb (Duskwatch Arcanist)
+		209404, -- Seal Magic (Duskwatch Arcanist)
+		{209516, "SAY", "FLASH"}, -- Mana Fang (Mana Saber)
+		224377, -- Drifting Embers (Infernal Imp)
+	}, {
+		[209027] = L.Guard,
+		[209036] = L.Sentry,
+		[225100] = L.Construct,
+		[211464] = L.Enforcer,
+		[211391] = L.Hound,
+		[214692] = L.Gerenth,
+		[207979] = L.Jazshariu,
+		[209378] = L.Imacutya,
+		[207980] = L.Baalgar,
+		[211299] = L.Inquisitor,
+		[211401] = L.BlazingImp,
+		[212031] = L.Energy,
+		[211470] = L.Mistress,
+		[209485] = L.Manifestation,
+		[209477] = L.Wyrm,
+		[209410] = L.Arcanist,
+		[209516] = L.Saber,
+		[224377] = L.InfernalImp,
+}
+end
+
+function mod:OnBossEnable()
+	self:RegisterMessage("BigWigs_OnBossEngage", "Disable")
+	-- Charging Station, Shadow Bolt Volley, Carrion Swarm, Shockwave, Whirling Blades, Drain Magic, Wild Detonation, Nightfall Orb, Seal Magic
+	self:Log("SPELL_CAST_START", "AlertCasts", 225100, 214692, 214688, 207979, 209378, 209485, 209477, 209410, 209404)
+	-- Quelling Strike, Fel Detonation, Searing Glare, Eye Storm, Drifting Embers, Charged Blast, Suppress, Charged Smash, Drifting Embers
+	self:Log("SPELL_CAST_START", "AlarmCasts", 209027, 211464, 211299, 212784, 211401, 212031, 209413, 209495, 224377)
+	-- Felblaze Puddle, Disrupting Energy
+	self:Log("SPELL_AURA_APPLIED", "PeriodicDamage", 211391, 209512)
+	self:Log("SPELL_PERIODIC_DAMAGE", "PeriodicDamage", 211391, 209512)
+	self:Log("SPELL_PERIODIC_MISSED", "PeriodicDamage", 211391, 209512)
+	-- Shadow Slash, Bewitch, Suppress, Mana Fang, Fortification, Cripple, Throw Torch
+	self:Log("SPELL_AURA_APPLIED", "DispellableDebuffs", 211473, 211470, 209413, 209516, 209033, 214690, 209036)
```

Comment:
Why not use a separate APPLIED function for it then instead of _START?

I notice you don't seem to use _SUCCESS much. Keep in mind that _START is only really useful for spells you need to avoid prior to being casted, or that need interrupted, etc.